src/views/open-api-docs-view/server.ts

```diff
@@ -106,6 +106,9 @@ export async function getStaticProps({
 	return {
 		props: {
 			productData,
+			title: schemaData.info.title,
+			releaseStage: targetVersion.releaseStage,
+			description: schemaData.info.description,
```

Comment:
Did add `status` yet since were not sure exactly where that data will come from.